app/components/candidate_interface/degree_grade_component.html.erb

```diff
@@ -3,7 +3,7 @@
   <%= form.govuk_error_summary %>
   <%= form.govuk_radio_buttons_fieldset :grade, legend: { text: legend_helper, size: 'l' }, hint: { text: t('application_form.degree.grade.international.grade_examples') } do %>
```

Comment:
Can we drop the hint text where there are radios buttons please? The hint text only makes sense where there's a free text box (ie for qualifications equivalent to a degree).